### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by kristebo** [scriptotek/bibrex#3]
It would be awesome if an item had an array for parts. This would be great for kits and multi item things.

**Comment by kristebo** [comment]
It is not allways an good option to tag every part of an kit, 
The usecase is arduinokits, some parts are usables, some are tiny and some are larger.

The checklist option is better.

**Comment by danmichaelo** [comment]
Right, a simple solution that could be used for different cases would be to have a (markdown) note field that pops up on returns (`check_in_reminder`?) would that work? Should it be for things, items or both?

**Comment by kristebo** [comment]
I think both.
